docs/design/2020-05-08-standardize-error-codes-and-messages.md

```diff
@@ -162,35 +162,64 @@ In addition, the error codes should be append only in case of conflict between v
 ### The Error Definition
 
 In the discussion above, an error has at least 4 parts:
-- The error code: it's the identity of an error
-- The error field: it's the error itself the user can view in TiDB system
-- the message field: the description of the error, what happened and why happend?
-- the workaround filed: how to workaround this error
+- The error code: it's the identity of an error.
+- The error field: it's the error itself the user can view in TiDB system. Like `err.Error()`.
+- The description field: the description of the error, what happened and why happened? This could be written by developer outside the code, and the more detail this field explaining the better, even some guess of cause could be included.
```

Comment:
```suggestion
- The description field: the expanded detail of why this error occurred. This could be written by developer outside the code, and the more detail this field explaining the better, even some guess of the cause could be included.
```